codeeve/settings.py

```diff
@@ -43,6 +43,7 @@
     'allauth.account',
     'allauth.socialaccount',
     'allauth.socialaccount.providers.github',
+    'debug_toolbar',
     'codeeve',
```

Comment:
instead of adding it always, you can do:
```
INSTALLED_APPS = ( ...)
if dev_env: # you can make a flag that is True in settings.py and False in production_settings.py, or use DEBUG
     INSTALLED_APPS += ('debug_toolbar')
```